### ISSUE TITLE
Inconsistant naming of Ctrlppcheck tool

### ISSUE THREAD

**Issue by mPokornyETM** [siemens/CtrlppCheck#91]
**Describe the bug**
The Ctrlppcheck is still inconsistent named.
Sometimes it is Cppcheck, sometimes Ctrlppchek

**To Reproduce**
Grep for cppcheck in the source code

![image](https://github.com/siemens/CtrlppCheck/assets/89339813/607de007-d2da-439c-8c86-96c6a58135ec)

**Expected behavior**
Ctrlppcheck shall be used instead of cppcheck

**Screenshots**
NN

**Software version and environment (please complete the following information):**
NN

**Additional context**
Check also help pls. In my try I checked only typical ctrl usage.

**Comment by Daradiridatumtarides** [comment]
in the tools directory there are three subdirs that contain classes that are "wrappers":
python
cppcheck
lizard

I propose naming them pythonW, ctrlppcheckW and lizardW.
also to be renamed: 
cppcheckerror -> ctrlppcheckerror
cppchecksettings -> ctrlppchecksettings

**Comment by mPokornyETM** [comment]
> pythonW, ctrlppcheckW and lizardW

What is the reason?

**Comment by Daradiridatumtarides** [comment]
didn't do it anyway

**Comment by mPokornyETM** [comment]
> didn't do it anyway

but, why shall be renamed? I does not see any reasons for that.

**Comment by mPokornyETM** [comment]
@Daradiridatumtarides do you want to continue here ? 
https://github.com/siemens/CtrlppCheck/commits/feature/91-inconsistent-naming-of-ctrlppcheck-tool/

**Comment by Daradiridatumtarides** [comment]
Yes, would like to. but I can not download a WinCC OA Version to test and continue.
(I have a login to WinCC OA portal)
